frontend/dashboard/context/HeaderContext/HeaderContext.test.tsx

```
@@ -0,0 +1,45 @@
+    renderWithProviders(
+      <HeaderContextProvider>
+        <button>My button</button>
+      </HeaderContextProvider>,
+    );
+
+    expect(screen.getByRole('button', { name: 'My button' })).toBeInTheDocument();
+  });
+
+  it('should provide a useHeaderContext hook', () => {
+    const TestComponent = () => {
+      const {} = useHeaderContext();
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)